packages/app/metrics-utils/README.md

```diff
@@ -0,0 +1,49 @@
+# Metrics Utils
+
+This package contains set of utilities to work with Prometheus metrics.
+
+Table of contents:
+1. [AbstractCounterMetric](#AbstractCounterMetric)
+
+### AbstractCounterMetric
+
+A base class for single counter-like metrics.
```

Comment:
can you add short explanation why you may prefer this over raw Prometheus metrics? e. g. safe multiple inits

Reply:
Sure, added an extra explanation there